python/setup.py

```diff
@@ -510,6 +510,11 @@ def _move_shared_libs_unix(build_prefix, build_lib, lib_name):
     lib_filename = os.path.basename(libs[0])
     shutil.move(pjoin(build_prefix, lib_filename),
                 pjoin(build_lib, 'pyarrow', lib_filename))
+    for lib in libs[1:]:
```

Comment:
Can you add a comment on what the purpose of this is?

Reply:
It's just reinstating code that was removed in ARROW-5082. The comment is just above ("Longest suffix library should be copied, all others symlinked").